### PR TITLE
Fix CTB reloading state

### PR DESCRIPTION
**Pull request by soupette**
Signed-off-by: soupette <cyril@strapi.io>

### What does it do?

It makes sure to wait for the server to have restarted before displaying the CTB.

### Why is it needed?

To improve the content modelling experience.

### How to test it?

Tests should be done in all 3 example apps (getstarted, kitchensink, kitchensink-ts)
1. Go to the ctb and create :
  - collection type
  - single type
  - a component
  A make sure it is working correctly
2. Do the same for PUT methods (add some field, edit some...)
3. Test the DELETE method for all created data structure.

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request

### CODE REVIEW

File: packages/core/admin/admin/src/reducers.js [hunk 1]
```diff
@@ -1,19 +1,21 @@
...
-  'content-manager_app': appReducer,
+  'content-manager_app': cmAppReducer,
   'content-manager_listView': listViewReducer,
   'content-manager_rbacManager': rbacManagerReducer,
   'content-manager_editViewLayoutManager': editViewLayoutManagerReducer,
   'content-manager_editViewCrudReducer': editViewCrudReducer,
 };
 
 const reducers = {
+  'strapi-admin_app': appReducer,
```
Comment (gu-stav) [inline packages/core/admin/admin/src/reducers.js:18]: Just a suggestion: no need to change it: What do you think about naming it `admin_app` instead? All the others aren't prefixed with strapi either and there isn't another admin ...

File: packages/core/content-type-builder/admin/src/components/DataManagerProvider/utils/request.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      method: 'HEAD',
+      mode: 'no-cors',
+      headers: {
+        'Content-Type': 'application/json',
+        'Keep-Alive': false,
+      },
+    })
+      .then(res => {
+        if (res.status >= 400) {
+          throw new Error('not available');
```
Comment (gu-stav) [inline packages/core/content-type-builder/admin/src/components/DataManagerProvider/utils/request.js:22]: Could we improve the wording of this error message? :)

**Review by gu-stav — APPROVED** [review]
I haven't tested it functionally, but the code looks good to. Two small suggestions, which you can ignore if you feel like they aren't that important.

**Review by Convly — CHANGES_REQUESTED** [review]
Only one small suggestion

**Review by Convly — APPROVED** [review]
LGTM

**Pull request merged**